tests/micropython/viper_ptr16_load.py

```diff
@@ -35,3 +40,27 @@ def memadd2(src_in) -> int:
 print(get(b), get1(b))
 print(memadd(b, 2))
 print(memadd2(b))
+
+# Test boundary conditions for various architectures
+
+import gc
+
+gc.collect()
```

Comment:
Is this `gc.collect()` necessary?

Actually I think it's best to split all these new tests out into separate files, eg `viper_ptr16_load_boundary.py`.  Because targets like esp8266 already have enough issues importing large-ish files with native code.

Reply:
The PR now has the boundary tests in their own files, thanks for the suggestion!